terragrunt/aws/api/iam.tf

```diff
@@ -129,6 +129,16 @@ data "aws_iam_policy_document" "api_policies" {
       aws_ssm_parameter.api_auth_token.arn
     ]
   }
+
+  statement {
```

Comment:
can we add a condition so that it can only assume roles in our aws org. i have the aws org id as a secret already in this repo.

```hcl
condition {
  test     = "StringEquals"
  values   = [var.aws_org_id]
  variable = "aws:PrincipalOrgID"
}
```

Reply:
ah right, totally forgot - I'll get that added